README.md

```diff
@@ -79,13 +72,15 @@ Fill in the following information:
   ```url
   https://raw.githubusercontent.com/IBM/ibm-streams-with-ml-model/master/notebooks/shopping_cart_kmeans_cluster_model.ipynb
   ```
-  ![new-notebook](doc/source/images/new-notebook.png)
+* Click the `Create notebook` button.
```

Comment:
you have numbered steps here for input data to create the notebook - did you want those numbers displayed in the appropriate fields in the image?